eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.js

```diff
@@ -104,24 +112,26 @@ const Menu = ({ data, onSelected, isSelected }) => {
       <Column width={56}>
         <ScrollPane>
           <MenuItemList>
-            {tabs[selectedTab].destinations.map((dest) => (
-              <SubMenuItem
-                key={dest.id}
-                aria-selected={isSelected(dest)}
-                onClick={() => onSelected(dest)}
-                tabIndex={0}
-                onKeyUp={(event) =>
-                  (event.key === Enter || event.key === Space) &&
-                  onSelected(dest)
-                }
-              >
-                <MenuItemTitles>
-                  <MenuItemTitle>
-                    <Truncated>{dest.displayName}</Truncated>
-                  </MenuItemTitle>
-                </MenuItemTitles>
-              </SubMenuItem>
-            ))}
+            {tabs[selectedTab].destinations.map((dest) =>
+              questionnaire.hub && dest.id === "EndOfQuestionnaire" ? null : (
+                <SubMenuItem
+                  key={dest.id}
+                  aria-selected={isSelected(dest)}
+                  onClick={() => onSelected(dest)}
+                  tabIndex={0}
+                  onKeyUp={(event) =>
+                    (event.key === Enter || event.key === Space) &&
+                    onSelected(dest)
+                  }
+                >
+                  <MenuItemTitles>
+                    <MenuItemTitle>
+                      <Truncated>{dest.displayName}</Truncated>
+                    </MenuItemTitle>
+                  </MenuItemTitles>
+                </SubMenuItem>
+              )
+            )}
```

Comment:
You could make this more reusable in the future by extending `.destination` to include an `enabled` or `disabled` toggle in it's JSON. Then, when mapping through each destination, you can instead build your logic around that.

The benefit of this is that the implementation wouldn't care what it is that's disabled or enabled, so in the future if we ever want to add more destinations to the disabled list we could modify the JSON instead of adding more logic.

Reply:
really good thinking - like it!